pyproject.toml

```diff
@@ -20,18 +19,24 @@ classifiers = [
 ]
 
 [tool.poetry.plugins."sceptre.resolvers"]
-"custom" = "resolver.custom:Custom"
 
 [tool.poetry.dependencies]
+types-PyYAML = "6.0.12.9"
 python = ">=3.8,<3.12"
+Jinja2 = "^3.1.2"
 
 [tool.poetry.group.dev.dependencies]
+mypy = "^1.2.0"
+pytest = "^7.3.0"
 pre-commit = "^3.2.1"
 tox = "^4.0.0"
-pytest = "^7.2.2"
 pytest-cov = "^4.0.0"
 sceptre = ">3.2"
+types-pyyaml = "^6.0.12.9"
 
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[virtualenvs]
+path = "venv"
```

Comment:
Why would you include this? This doesn't look like it's necessary. Poetry works with all sorts of different virtual environment frameworks.

Reply:
done.